src/components/definition-list/definition-list.test.tsx

```diff
@@ -0,0 +1,257 @@
+    </Dl>
+  );
+
+  expect(screen.getByTestId("dt")).toHaveStyleRule(
+    "margin-bottom: var(--spacing300)"
+  );
+  expect(screen.getByTestId("dt")).toHaveStyleRule(
+    "padding-right: var(--spacing500)"
+  );
+});
+
+test("the expected default styles are applied to the Dl element", () => {
+  render(
+    <Dl>
+      <Dt>Title</Dt>
+      <Dd>Description</Dd>
+    </Dl>
+  );
+
+  expect(screen.getByTestId("dl")).toHaveStyle({
+    display: "grid",
+    height: "auto",
+    width: "100%",
+    backgroundColor: "transparent",
+    gridTemplateRows: "auto",
+    gridTemplateColumns: "50% auto",
+  });
+});
+
+test("the expected default styles are applied to the Dt element", () => {
+  render(
+    <Dl>
+      <Dt>Title</Dt>
+      <Dd>Description</Dd>
+    </Dl>
+  );
+
+  const dtElement = screen.getByTestId("dt");
+
+  expect(dtElement).toHaveStyleRule("fontSize: var(--fontSizes100)");
+  expect(dtElement).toHaveStyleRule("color: var(--colorsUtilityYin090)");
+  expect(dtElement).toHaveStyleRule("paddingRight: var(--spacing300)");
+  expect(dtElement).toHaveStyleRule("marginBottom: var(--spacing200)");
+});
+
+test("the expected default styles are applied to the Dd element", () => {
+  render(
+    <Dl>
+      <Dt>Title</Dt>
+      <Dd>Description</Dd>
+    </Dl>
+  );
+
+  expect(screen.getByTestId("dd")).toHaveStyle("margin-left: 0px");
+  expect(screen.getByTestId("dd")).toHaveStyleRule(
+    "font-size: var(--fontSizes100)"
+  );
+  expect(screen.getByTestId("dd")).toHaveStyleRule(
+    "font-weight: var(--fontWeights400)"
+  );
+  expect(screen.getByTestId("dd")).toHaveStyleRule(
+    "color: var(--colorsUtilityYin065)"
+  );
+  expect(screen.getByTestId("dd")).toHaveStyleRule(
+    "margin-bottom: var(--spacing200)"
+  );
+});
+
+test("allows a custom width percentage to be applied to the Dl element via the `w` prop", () => {
+  render(
+    <Dl w={45}>
+      <Dt>Title</Dt>
+      <Dd>Description</Dd>
+    </Dl>
+  );
+
+  expect(screen.getByTestId("dl")).toHaveStyle(
+    "grid-template-columns: 45% auto"
+  );
+});
+
+test("allows custom padding and margin styles to be applied to the Dt element", () => {
+  render(
+    <Dl>
+      <Dt mb={1} pr={2}>
+        Title
+      </Dt>
+      <Dd>Description</Dd>
+    </Dl>
+  );
+
+  const dtElement = screen.getByTestId("dt");
+
+  expect(dtElement).toHaveStyleRule("paddingRight: var(--spacing200)");
+  expect(dtElement).toHaveStyleRule("marginBottom: var(--spacing100)");
+});
+
+test("allows custom margin-bottom styles to be applied to the Dd element via the `mb` prop", () => {
+  render(
+    <Dl>
+      <Dt>Title</Dt>
+      <Dd mb={1}>Description</Dd>
+    </Dl>
+  );
+
+  expect(screen.getByTestId("dd")).toHaveStyleRule(
+    "margin-bottom: var(--spacing100)"
+  );
+});
+
+describe.each(["left", "center", "right"] as ElementAlignment[])(
+  "when the align prop is set to %s",
+  (align) => {
+    it("matches the custom text alignments passed to Dt", () => {
```

Comment:
**suggestion** (non-blocking): since we're moving against using nested `describe`s, I wonder if this would be better with two different `it.each` blocks rather than 2 `it`s inside a `describe.each`.